doc/man3/SSL_get_version.pod

```diff
@@ -25,10 +25,14 @@ initiating the connection.
 SSL_get_version() returns the name of the protocol used for the
 connection.
 
-SSL_is_dtls() returns one if the connection is using DTLS, zero if not.
-
 SSL_version() returns the protocol version used for the connection.
 
+The above functions should only be called after the intial handshake has been
```

Comment:
I'm worried about a future ambiguity with "The above"  How about merging the three paragraphs, delete lines 26 and 29 and say "These" instead of "The above" ?

Reply:
... or "They"

Reply:
Done!